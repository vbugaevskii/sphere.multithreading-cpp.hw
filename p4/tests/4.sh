echo blabla > file.txt
echo aaa2 > file1.txt
cat file.txt file.txt file.txt file1.txt | sort | uniq

false | true && echo 1-true
true | false || echo 2-false

