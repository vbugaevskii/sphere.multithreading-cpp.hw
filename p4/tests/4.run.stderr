Process 13951 exited: 0
Process 13952 exited: 0
Process 13953 exited: 0
Process 13954 exited: 0
Process 13955 exited: 0
Process 13956 exited: 1
Process 13957 exited: 0
Process 13958 exited: 0
Process 13959 exited: 0
Process 13960 exited: 1
Process 13961 exited: 0
